GPU/Common/SplineCommon.cpp

```diff
@@ -361,16 +383,45 @@ static void _SplinePatchFullQuality(u8 *&dest, u16 *indices, int &count, const S
 	}
 }
 
+template <bool origNrm, bool origCol>
+static inline void SplinePatchFullQualityDispatch3(u8 *&dest, u16 *indices, int &count, const SplinePatchLocal &spatch, u32 origVertType, int quality, int maxVertices) {
+	bool origTc = (origVertType & GE_VTYPE_TC_MASK) != 0;
```

Comment:
I used VerySleepy a lot because it's convenient and I hooked it up with the jit.  The VS profiler is also pretty good, and so is the Android profiler (although not sure if it still works after all the build script changes.)

VerySleepy samples, so sometimes it can be inaccurate and you have to make something run over and over again to get good stats.

The other two mark up function in/out, but this can add a little bias since each function call is doing a bit of extra work.  Usually it's minor, though.

-[Unknown]